src/store/index.js

```diff
@@ -153,10 +153,10 @@ export class ApplicationStore {
 
     @computed
     get filterDiscoveryYear() {
-        const cutoff = this.year;
+        const [low, high] = this.year;
         return (node) => {
-            const year = node.discovery;
-            return (year == null && cutoff === 2016) || (year != null && year <= cutoff);
+            const discovery = node.discovery;
```

Comment:
instead of the 2016, could you use `this.yearRange[1]`?

Reply:
Good catch, thanks. See 99cc9f2017a33aec09c5a8b71969ac9941c7fce1.